openapi_configuration.yml

```diff
@@ -919,6 +1077,248 @@ components:
       required:
         - versions
       additionalProperties: false
+    PostGoldenExtraction:
+      type: object
+      properties:
+        golden:
+          type: string
+          description: user-friendly name for the reference document #question: seems inconsistent that other parts of the internal API require doc type ID, not userfriendly name
```

Comment:
"reference document" means golden, not document type. we only use IDs for document types

Reply:
The question might be misplaced, but in general, it stands: we use document type ID in most endpoints other than extarct from golden endpoints, where we document type name. Is that because the extract golden endpoints don't have any post operations for document types so we care less if they screw up which type they're using?